blog-cse/2022-10-13-application-update.md

```diff
@@ -0,0 +1,86 @@
+  <tr><td>lanScanner</td><td>lan_scanner_exception_ips</td></tr>
+  <tr><td>nms</td><td>nms_ips</td></tr>
+  <tr><td>paloAltoSinkhole</td><td>palo_alto_sinkhole_ips</td></tr>
+  <tr><td>proxyServer</td><td>proxy_servers<br/>proxy_servers_dst<br/>proxy_servers_src</td></tr>
+  <tr><td>vpnServer</td><td>vpn_servers</td></tr>
+  <tr><td>vulnerabilityScanner</td><td>vuln_scanners</td></tr>
+  <tr><td>webServer</td><td>http_servers</td></tr>
+  <tr><td rowspan="3">_networkType</td><td>guest</td><td>guest_networks</td></tr>
+  <tr><td>nat</td><td>nat_ips</td></tr>
+  <tr><td>vpn</td><td>vpn_networks</td></tr>
+  <tr><td rowspan="6">_userGroup</td><td>awsAdmin</td><td>AWS_admin_users</td></tr>
+  <tr><td>dsReplication</td><td>ds_replication_authorized_users</td></tr>
+  <tr><td>gcpAdmin</td><td>GCP_admin_users</td></tr>
+  <tr><td>googleWorkspaceAdmin</td><td>Google_Workspace_admin_users</td></tr>
+  <tr><td>kerberosDowngrade</td><td>downgrade_krb5_etype_authorized_users</td></tr>
+  <tr><td>salesforceAdmin</td><td>salesforce_admin_users</td></tr>
+</table>
+
+(There are five standard match lists *not* affected by this change, as they do not contain Entities. These include: business_asns, business_domains, business_hostnames, threat, and verified_uri_paths.)
+
+**Beginning Thursday, October 20** the contents of the standard match lists listed above will automatically be copied to tags set on the individual entities. So, for example, if an Entity `1.2.3.4` is in match list `sql_servers`, a tag `_deviceService:sql` will be set on it. CSE will continue to automatically create these tags from the standard match lists for a period of 3 months, **until January 20, 2023**. During this period, pre-defined rules will be updated to reference these tags instead of the standard match lists, so by the end of this period all rules will be updated and CSE will no longer automatically create these tags.
+
+**Please update any process you use to maintain the members of standard match lists by January 20, 2023 to maintain standard Entity tags instead (or in addition).** We highly recommend you take advantage of Entity Groups to set Entity tags rather than individually setting tags. Entity Groups enable the automatic application of attributes like tags based on the Entity's value, IP address range, or inventory group. 
+
+Note that you cannot extend the standard tag schemas (for example, you cannot add a value `azureAdmin` to `_userGroup`). (The underscore prefix in the schema name means it's a system-defined schema.) Instead, create a different tag schema (such as `customUserGroup`) with such extended values. 
+
+You can refer to Entity tags in Rule expressions. For example, if you've attached the tag `_deviceService:sql` to an Entity, 
+this statement will return "true" if that Entity is listed in a Record's `srcDevice_ip` field: 
+
+    array_contains(fieldsTags["srcDevice_ip"], "_deviceService:sql")
```

Comment:
I'd put backticks around this, so it's displayed as code: array_contains(fieldsTags["srcDevice_ip"], "_deviceService:sql")